src/pages/AdminSettings/AdminMembers/MemberList/MemberTable.tsx

```diff
@@ -94,6 +93,28 @@ const createTable = ({ tableData, seatData, mutate }: CreateTableArgs) => {
   )
 }
 
+interface Params {
+  activated?: boolean
+  search?: string
+  isAdmin?: boolean
+}
+
+const useQueryMembersList = (params: Params) => {
+  const { data: queryData, ...rest } = useInfiniteQueryV5(
+    SelfHostedUserListQueryOpts(params)
+  )
+
+  const flatUsersList = useMemo(
+    () => queryData?.pages?.flatMap((page) => page.results) ?? [],
+    [queryData?.pages]
+  )
+
+  return {
+    ...rest,
```

Comment:
any chance we can avoid using ...rest here? Do we really need everything coming back from this query call?